backend/src/main/java/edu/netcracker/backend/dao/impl/PossibleServiceDAOImpl.java

```diff
@@ -0,0 +1,76 @@
+package edu.netcracker.backend.dao.impl;
+
+import edu.netcracker.backend.dao.PossibleServiceDAO;
+import edu.netcracker.backend.dao.ServiceDAO;
+import edu.netcracker.backend.model.PossibleService;
+import edu.netcracker.backend.model.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class PossibleServiceDAOImpl extends CrudDAOImpl<PossibleService> implements PossibleServiceDAO {
+
+    private ServiceDAO serviceDAO;
+
+    private String FIND_ALL_WITH_CLASS_ID = "SELECT * FROM possible_service " +
+            "WHERE class_id = ?";
+
+    private String FIND_ALL_P_SERVICES_BY_SUGGESTION_ID = "SELECT * FROM possible_service " +
```

Comment:
Are you sure that you need to use * with JOINs?

Reply:
I agree, but it works. I promise this will optimize in the future :)